specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -42,7 +40,42 @@ import {
   HeaderItemsPreferenceItem,
   WelcomePageModePreferenceItem,
 } from './Renderers';
-import { defineItem, GenericPreferences } from './types';
+import {
+  defineItem,
+  GenericPreferences,
+  PreferenceItem,
+  PreferencesVisibilyContext,
+} from './types';
+import { StatLayout } from '../Statistics/types';
+import { statsText } from '../../localization/stats';
+
+// Custom Renderer for a preference item
+export type PreferenceItemComponent<VALUE> = (props: {
```

Comment:
This is not merged correctly. PreferenceItemComponent was removed from this file, but added back accidentally when resoling merge conflicts
I will push a fix to production for this